docs/azure_arc_jumpstart/azure_arc_k8s/iot_uses_cases/aks/_index.md

```diff
@@ -0,0 +1,243 @@
+
+    ![Screenshot showing terraform plan completing](./02.png)
+
+    ![Screenshot showing Azure Portal with AKS resource](./03.png)
+
+    ![Screenshot showing Azure Portal with AKS resource](./04.png)
+
+
+* In this scenario we will use a VM to "simulate" an IoT Edge device. To do this, we must register a new Edge device in our IoT Hub that we will later configure. 
+
+* In order to keep your local environment clean and untouched, we will use [Azure Cloud Shell](https://docs.microsoft.com/en-us/azure/cloud-shell/overview) (located in the top-right corner in the Azure portal) to run the next commands:
+
+    ![Screenshot showing how to access Cloud Shell in Visual Studio Code](./05.png)
+
+* Create IoT Edge Device
+ ```shell
+az iot hub device-identity create --device-id "EdgeDeviceSim" --edge-enabled --hub-name k8sedgejumpstart
+ ```
+
+> * We will obtain the connection string of the new IoT Edge device to be able to make the link
+```shell
+az iot hub device-identity connection-string show --device-id "EdgeDeviceSim" --hub-name k8sedgejumpstart
+  ```
+
+  ![Screenshot showing how to access Cloud Shell in Visual Studio Code](./06.png)
+
+  ![Screenshot showing how to access Cloud Shell in Visual Studio Code](./07.png)
+
+* Modify the file _/etc/aziot/config.toml_ in the VM that we have deployed using your SSH credentials.
+Once inside the VM, we will modify the /etc/aziot/config.toml file including the following lines at the end of the file with the value of the connection string that we have obtained in the previous step: 
+
+```shell
+# Manual provisioning with connection string
+[provisioning]
+source = "manual"
+connection_string = "<ADD DEVICE CONNECTION STRING HERE>"
+ ```
+
+* In order to synchronize the configuration of the device that we have paired we must execute the following command:
+
+```shell
+sudo iotedge config apply
+ ```
+
+![Screenshot showing sync config IoT Edge device in VM](./08.png)
+
+* Once completed the above steps, return to the Azure Cloud Shell where we will assign to our new device a module to simulate a temperature sensor. For this we will upload the file through the Azure Cloud Shell interface:  
+
+![Screenshot showing sync config IoT Edge device in VM](./09.png)
+
+* Once the file is uploaded, execute the following command: 
+
+> **Note: You can see an example of the [deployment.json](https://github.com/main/Kubernetes_edge/blob/main/deploy/scripts/edge/deployment.json) file that we use.**
+
+```shell
+az iot edge set-modules --hub-name k8sedgejumpstart --device-id "EdgeDeviceSim" --content ./deployment.json
+```
+
+* From the Azure portal, select the IoT Hub instance under _K8sEdgeJumpStart_. By selecting our IoT Edge device, we can see all the information about the modules it is running and If everything has been successful we will see that the "SimulatedTemperatureSensor" module is running correctly.
+
+![Screenshot showing IoT Edge device in IoT Hub](./10.png)
```

Comment:
Add step numbers, no need for arrows.

Reply:
Perfect! I upload a new version adding numbers !! ^^